substrate/frame/treasury/src/lib.rs

```diff
@@ -122,6 +122,8 @@ pub type NegativeImbalanceOf<T, I = ()> = <<T as Config<I>>::Currency as Currenc
 >>::NegativeImbalance;
 type AccountIdLookupOf<T> = <<T as frame_system::Config>::Lookup as StaticLookup>::Source;
 type BeneficiaryLookupOf<T, I> = <<T as Config<I>>::BeneficiaryLookup as StaticLookup>::Source;
+pub type BlockNumberFor<T, I = ()> =
```

Comment:
not sure if we should use same naming. we have `BlockNumberFor` reference type in prelude that we use to reference the number from the header.